frontend/src/employee-frontend/components/reports/VoucherServiceProviderUnit.tsx

```diff
@@ -123,56 +162,110 @@ function VoucherServiceProviderUnit() {
               }
               styles={reactSelectStyles}
             />
-          </Wrapper>
+          </FilterWrapper>
         </FilterRow>
 
-        {rows.isLoading && <Loader />}
-        {rows.isFailure && <span>{i18n.common.loadingFailed}</span>}
-        {rows.isSuccess && (
+        { report.isSuccess && report.value.locked && (
+          <LockedDate spacing='xs' alignItems='center'>
+            <FontAwesomeIcon icon={faLockAlt}/>
+            <span>
+              {`${i18n.reports.voucherServiceProviders.locked}: ${report.value.locked.format()}`}
+            </span>
+          </LockedDate>
+        )}
+
+        <HorizontalLine slim/>
+
+        {report.isLoading && <Loader />}
+        {report.isFailure && <span>{i18n.common.loadingFailed}</span>}
+        {futureSelected && <>
+          <Gap />
+          <InfoBox wide message={i18n.reports.voucherServiceProviders.filters.noFuture}/>
+        </>}
+        {!futureSelected && report.isSuccess && (
           <>
-            <ReportDownload
-              data={rows.value}
-              headers={[
-                {
-                  label: i18n.reports.voucherServiceProviderUnit.childFirstName,
-                  key: 'childFirstName'
-                },
-                {
-                  label: i18n.reports.voucherServiceProviderUnit.childLastName,
-                  key: 'childLastName'
-                },
-                {
-                  label: i18n.reports.common.groupName,
-                  key: 'childGroupName'
-                },
-                {
-                  label: i18n.reports.voucherServiceProviderUnit.numberOfDays,
-                  key: 'numberOfDays'
-                },
-                {
-                  label:
+            <SumRow>
+              <FixedSpaceRow>
+                <strong>{`${i18n.reports.voucherServiceProviderUnit.total}`}</strong>
+                <strong>{formatCents(report.value.voucherTotal)}</strong>
+              </FixedSpaceRow>
+
+              <ReportDownload
+                data={[
+                  ...report.value.rows.map(r => ({
+                    ...r,
+                    start: r.realizedPeriod.start.format(),
+                    end: r.realizedPeriod.end.format(),
+                    note: r.type === 'REFUND' ? 'Hyvitys' : r.type === 'CORRECTION' ? 'Korjaus' : '',
+                    serviceVoucherServiceCoefficient: (r.serviceVoucherServiceCoefficient / 100.0).toFixed(2).replace('.', ','),
```

Comment:
Why not use the function `formatCents` here? I know technically they're not "cents", but they're both hundredth's and you're trying to represent them in the same way.